packages/nimble-components/src/table/tests/table-matrix.stories.ts

```diff
@@ -29,20 +29,29 @@ export default metadata;
 const data = [
```

Comment:
Do you expect we'll change this data back to numerics and booleans once we have column types for those? To me it's a little weird to have fileds named `myNum` and `myBool` which are strings. It can also give a false impression that we're testing type conversion logic or something, but really it's just more string data.

Same comment for the non-matrix test.

Reply:
I've changed the table data to be text-centric.